packages/dds/map/src/directory.ts

```diff
@@ -1203,6 +1207,7 @@ class SubDirectory extends TypedEventEmitter<IDirectoryEvents> implements IDirec
 		super();
 		this.localCreationSeqTracker = new DirectoryCreationTracker();
 		this.ackedCreationSeqTracker = new DirectoryCreationTracker();
+		this.mc = loggerToMonitoringContext(this.logger);
```

Comment:
if the change isn't too big can we remove logger as a member, and only use mc. having multiple ways to log can cause problems.